engine/src/main/java/org/camunda/bpm/engine/impl/core/CoreLogger.java

```diff
@@ -58,14 +61,6 @@ public ProcessEngineException duplicateVariableInstanceException(CoreVariableIns
       ));
   }
 
-  public ProcessEngineException missingVariableInstanceException(CoreVariableInstance variableInstance) {
-    return new ProcessEngineException(exceptionMessage(
-        "005",
-        "Cannot update variable instance with name {}. Variable does not exist",
-        variableInstance.getName()
-      ));
-  }
```

Comment:
Maybe a comment to prevent using the code, the log is not used in previous versions as well. Let's discuss it briefly.

Reply:
👍 I'll add a comment and backport it down to our maintained versions so that nobody is tempted to reuse the identifier.